ui/pages/password-reset/index.js

```diff
@@ -80,44 +50,7 @@ export default function PasswordReset() {
               <div className='w-full border-t border-gray-800' />
             </div>
           </div>
-          <form
-            onSubmit={onSubmitSetPassword}
-            className='relative flex w-full max-w-sm flex-col'
-          >
-            <div className='my-2 w-full'>
-              <label
-                htmlFor='password'
-                className='text-3xs uppercase text-gray-500'
-              >
-                Password
-              </label>
-              <input
-                required
-                id='password'
-                type='password'
-                data-testid='form-field-password'
-                placeholder='enter your password'
-                onChange={e => {
-                  setPassword(e.target.value)
-                  setError('')
-                }}
-                className={`w-full border-b border-gray-800 bg-transparent px-px py-2 text-2xs placeholder:italic focus:border-b focus:outline-none focus:ring-gray-200 ${
-                  error ? 'border-pink-500/60' : ''
-                }`}
-              />
-            </div>
-            <button
-              disabled={!password}
-              className='mt-6 mb-2 rounded-lg border border-violet-300 px-4 py-3 text-2xs text-violet-100 hover:border-violet-100 disabled:pointer-events-none disabled:opacity-30'
-            >
-              Set Password
-            </button>
-            {error && (
-              <p className='absolute -bottom-3.5 mx-auto w-full text-center text-2xs text-pink-400'>
-                {error}
-              </p>
-            )}
-          </form>
+          <PasswordResetForm />
```

Comment:
is the component not in this PR?

Reply:
I missed two files, ui/pages/accept-invite/index.js and ui/components/password-reset-form.js. added them now

Reply:
I think the import is missing in this file